docs/actions/http.md

```diff
@@ -1,9 +1,288 @@
 # HTTP
 
+## Global Notes
+
+### Security
+  All the HTTP tasks uses the same security logic.
+  It is defined via the **target** input properties :
+  * Basic Authentication
```

Comment:
Beware, this bullet point list does not render properly.
To test your work locally, you can run `mkdocs serve` and check rendering on http://localhost:8000

You can follow installation for mkdocs: https://www.mkdocs.org/getting-started/
And then for material theme: https://squidfunk.github.io/mkdocs-material/getting-started/